### DISCUSSION TITLE
Who wants to be hired? (Nov 2021)

### DISCUSSION THREAD

**Discussion by Nutlope** [vercel/next.js#31541] [Ideas]
# What is this?
This discussion is for Next.js developers looking for work.

# Candidates

Share your information if you are looking for work. Please use this format:

  Location:
  Remote:
  Willing to relocate:
  Technologies:
  Résumé/CV:
  Website:
  Email:

Good luck with interviewing!

---

If you're a company looking to post a role, use #31542.

**Comment by Jassi10000-zz** [comment]
Location : New Delhi , India
Remote : Yes
Willing to relocate : Yes
Technologies : React.js , Node.Js , MongoDb, Express.Js , MySQL , Familiarity with Next.Js, TailwindCss , Git
Resume / CV : https://drive.google.com/file/d/1FM5kmXJesN7XR2rAySPuX0QN1o8mwLxq/view?usp=sharing
Website : https://jaskeeratsingh.netlify.app
Email : jaskeerat10000@gmail.com

**Reply by Jassi10000-zz** [reply]
@Nutlope Sir , are there any updates on our infos , No one has contacted us regaring the jobs , its been very long since we have posted our profile here with resumes and other stuff

**Reply by Nutlope** [reply]
Nope. I cannot guarantee someone will contact you. Some employers look at this page and reach out to those they are interested in.

**Reply by pratik-dani** [reply]
@Jassi10000-zz, are you still available? I am looking for frontend developer intern.

**Comment by MichaelFrieze** [comment]
Location: Clinton Township, MI, USA
Remote: Yes
Willing to relocate: Maybe
Technologies: JS, React, Next, Gatsby, CSS, Node/Express, Java, Swift
Résumé/CV: https://drive.google.com/file/d/1frLY2ebxKNMudDrk0vBPu9eqaUhs6ONL/view?usp=sharing
Website: https://www.michaelfrieze.dev/
Email: contact@michaelfrieze.dev

**Comment by chemiadel** [comment]
Location: Algeria
Remote: Yes
Willing to relocate: Yes
Technologies: JS/TS, ReactJS, NodeJS, NextJS, Graphql, ExpressJS, Firebase-admin, Deno
Résumé/CV: https://my.indeed.com/p/adelc-s89zs7b
Website: https://adel-portfolio.vercel.app/
Email: pace3man@gmail.com || adelchemi6@gmail.com

**Comment by joelmatiasduran** [comment]
Hello everyone!! 

Location: Buenos Aires, Argentina
Remote: Yes
Willing to relocate: Yes
Technologies: Next Js, React, TypeScript, JavaScript, Node Js, Nest Js, SQL, Mongodb 
Résumé/CV: https://docs.google.com/document/d/1GQwT9aZZwgDa2KHPZdUu_Xk1tiYKPlnRXQgGlhseO0E/edit?usp=sharing
Website: https://joelmatias.netlify.app/
Email: jmatiimartinez@gmail.com

**Reply by kailoon** [reply]
Location: 
**Malaysia**

Remote: 
**Yes**

Willing to relocate: 
**No**

Technologies: 
**nextjs, svelte, prisma, planetscale, css/scss/tailwindcss**

Résumé/CV: 
**https://github.com/kailoon/resume**

Website: 
**https://kailoon.com**

Email:
**chankailoon@gmail.com**

**Comment by kellenwiltshire** [comment]
Location: Ontario Canada
Remote: Yes
Willing to relocate: No
Technologies: NextJS, React, TailwindCSS, Strapi, PostgresSQL, Squareup/Shopify
Résumé/CV: [Linkedin](https://www.linkedin.com/in/kellenwiltshire/ )
Website: [kellenwiltshire.com](https://kellenwiltshire.com)
Email: webdevelopment@kellenwiltshire.com

**Reply by ardiewen** [reply]
Hi @kellenwiltshire, just giving you a heads up that I'll be sending you an email. Looking for some local-ish (I'm in Toronto) freelance help on some projects. Cheers!

**Reply by kellenwiltshire** [reply]
Awesome! I look forward to your email!

[a comment by an automated account is not shown]

**Comment by booleanrecep** [comment]
Location: Istanbul / Turkey
Remote: Yes
Willing to relocate: Yes
Technologies: React, Next,TypeScript,Scss,Redux
Résumé/CV:https://docs.google.com/document/d/1bCS7AmJTvTgb57JCKVafu1zxq0wFMInNsKAZZpEb9LY/edit?usp=drivesdk
Website:https://ahnung.vercel.app
Email: recep.fed@gmail.com

**Reply by EvelynBar** [reply]
We are looking for a Senior JavaScript Developer (Front End) let me know if you would be interested in discussing further. My email is evelyn.vargas@userway.org

**Comment by Aagam-Jain-113** [comment]
Location: India
Remote: Yes
Willing to relocate: Yes
Technologies: Next.js, Tailwind CSS, React.js, Node.js, JavaScript, MongoDB, Google Cloud Platform (GCP), 
Résumé/CV: https://drive.google.com/drive/folders/1YCHlFgKf4D__4DD2Vi5IDetZ5lDwWzXL?usp=sharing
Website: https://aagamjain.vercel.app/
Email: aagamjain113@gmail.com

**Comment by alketii** [comment]
Location: Pristina, Kosovo
Remote: Yes
Willing to relocate: Yes
Technologies: Next.js, Tailwind CSS, React, Redux, NodeJs, Strapi, GraphQL
Résumé/CV:
Website:
Email: alketii@gmail.com

**Reply by EvelynBar** [reply]
We are looking for a Senior JavaScript Developer (Front End) let me know if you would be interested in discussing further. My email is evelyn.vargas@userway.org

**Comment by danestves** [comment]
Location: Venezuela
Remote: Yes
Willing to relocate: Yes
Technologies: React/Next/Gatsby | Vue/Nuxt | Node/GraphQL/MySQL/MongoDB | GraphCMS/Strapi/Contenful/Sanity
Résumé/CV: [https://read.cv/danestves](https://read.cv/danestves) | [https://danestves.com/danestves.pdf](https://danestves.com/danestves.pdf)
Website: [https://danestves.com](https://danestves.com)
Email: danestves@gmail.com

**Reply by EvelynBar** [reply]
We are looking for a Senior JavaScript Developer (Front End) let me know if you would be interested in discussing further. My email is evelyn.vargas@userway.org

**Comment by Rolemodel01291** [comment]
[FOR HIRE][REMOTE]
I am a self-motivated 🎨Frontend Developer🎨, passionate about learning ❤️new technologies❤️. 
I have a working proficiency in JavaScript, React, Node, Express, PGSQL, MongoDB, Solidity smart contract. I have a solid understanding of Git version control.
Here are my skills.


🥇Main Skills:
✪ Smart contract, web3, Defi and Dapp, Multi-DEX on BSC, Polygon, Ethereum
✪ Frontend: React/Redux/Next.js/ Vue.js/ React Native
✪ Backend: Node.js/ Python/Django/ PHP/ Laravel/ RESTful API
✪ Database: MongoDB/ Fire store/ MySQL/ PostgreSQL
✪ AWS Services: EC2/ Lambda, S3/ DynamoDB
✪ Video Streaming/Conference: AWS Kinesis/ WebRTC/ Wowza/ Jitsi
✪ Project Management: GitHub / GitLab/ JIRA/ Trello/ Zoho

Please DM.

**Reply by fabien-huet** [reply]
Hi, is there a way to contact you ?

**Reply by EvelynBar** [reply]
We are looking for a Senior JavaScript Developer (Front End) let me know if you would be interested in discussing further. My email is evelyn.vargas@userway.org

**Comment by monire91** [comment]
I'm a mid-level front end developer.
Location: Iran
Remote: Yes
Willing to relocate: Yes
Technologies: Next.js/SCSS/GIT/RESTful API
Résumé/CV: https://drive.google.com/file/d/13TLOjK-C0FvY00MAc6A061XN2HUZFJeP/view?usp=sharing
Website: -
Email: aglida1370@gmail.com

**Comment by erinoggz** [comment]
Location: Lagos, Nigeria
Remote: Yes
Willing to relocate: Yes
Technologies: React / Next / Node / Nest / GraphQL / RESTful API / MySQL / MongoDB / Docker / Kubernetes / GCP
Résumé/CV: https://drive.google.com/file/d/1oXKfEa6QHfCd2hrc18udSsgRf4046FOI/view?usp=sharing
Website: https://erin.vercel.app/
Email: erin.deji@gmail.com

**Comment by rbourdon** [comment]
🗺️  Location: **Salem, Massachusetts USA**
📞 Remote: **Yes**
✈️Willing to relocate: **No**

🔧 Technologies: **Next.js - React - Framer Motion - GraphQL - PostgreSQL - Sendgrid - GraphCMS - Illustrator - Blender**

📄 Résumé/CV: **https://www.rorybourdon.com/Rory_Bourdon_Resume_2021.pdf**
🌐 Website: **https://www.rorybourdon.com/**
📧 Email: **rorybourdon@gmail.com**

**Comment by davidtran** [comment]
- Location: Vietnam GMT+7
- Remote: Yes
- Willing to relocate: No
- Technologies: ReactJS, NextJS, NodeJS, AWS, styled-components, GraphQL
- Portfolio: https://jslancer.com/portfolio/portfolio-2021.pdf
- Website: https://jslancer.com
- Email: david@jslancer.com

**Reply by EvelynBar** [reply]
I just sent you an email :)

**Comment by alvinrumbaoa** [comment]
Share your information if you are looking for work. Please use this format:

Location: Orange County/ Los Angeles County, California
Remote: Yes
Willing to relocate: No
Technologies: NextJs, Vercel, ChakraUI, MongoDB, Express, ReactJS, Framer-motion, NodeJs
Website: https://www.alvinrumbaoa.dev/
Email: alvinrumbaoa@gmail.com

**Comment by kumarsandeep91** [comment]
Location: Orai, India
Remote: Yes
Willing to relocate: No
Technologies: React, Redux, Next Js, HTML, CSS, Js
Résumé/CV: https://drive.google.com/file/d/14AKdpWrDmqLFWvrSIPxS5a_ghe5bWVtY/view?usp=sharing
Website: https://sandeepsihari.com
Email: kumarsandeep91@gmail.com

**Comment by dougwithseismic** [comment]
[FE Next.js Developer - Looking to pivot into Development from Marketing Consulting]

I'm twelve years into a Performance Marketing / Growth career; looking to pivot into FE development for some experience as a team.
Mosty recently built, grew and sold vouchernaut.com; a fully automated voucher site with zero overheads. Ported it over to NextJS from Gatsby last week.

Location: Prague, CZ / London UK
Remote: Yes
Willing to relocate: to London, UK
Technologies: React. Gatsby. Nextjs. Node. Express. JavaScript. Framer Motion.
Skills: UI design. Martech. Paid Media / SEM / SEO.  
Résumé/CV: [Doug Silkstone - Notion](https://dougwithseismic.notion.site/dougwithseismic/Doug-Silkstone-Performance-Marketing-User-Acquisition-JavaScript-Developer-0718e15bf95a4d6393e8579f686cfbf8)
Email:doug@withseismic.com

Hit me up for any remote work, please!

**Comment by idirmosh** [comment]
Somewhat full-stack developer looking for work opportunity, I am somewhat very comfortable with the react eco-system with a passion of learning new things.

**Location**: Greenacres, Florida, US
**Remote**: Yes
**Willing to relocate**: NO
**Technologies**: React. Nextjs. Node. Express. GraphQl, JavaScript. all css-in-js tech. 
**Design**: I have a good eye for user interfaces and web design in general.
**Résumé/CV**: I am currently working on the cv just moved to us not too long ago, currently working on getting the GED.
**Projects**: currently working on an issue-tracker app, the source can be found on my github :https://github.com/idirmosh/issuestracker, its still work on progress, hopefully it would be a demo of my coding skills.
**Website**: idirmosh.dev
**Email**:  idir.mosh@gmail.com
**LinkedIn**: https://www.linkedin.com/in/idirmosh/

Hit me up for any remote work, I would appreciate any opportunity.

**Comment by amar-jay** [comment]
Location: Istanbul, Turkey

Remote: Yes

Willing to relocate: No

Technologies:
👉 Frontend: React/Nextjs, JavaScript/TypeScript, Tailwind CSS/Chakra UI,
👉 Backend: Go, Gorm, Graphql(Gqlgen), gin
👉 Database: MongoDB, PostgreSQL, Firebase
👉 Payment: Stripe, flutterwave
👉 Cloud service: AWS, Google Cloud, Vercel
👉 CMS: WordPress, Wix.com, Sanity
👉 Testing: Jest, go-test, Storybook
👉 Languages: JS/TS, Go

Website: https://themanan.me/
LinkedIn: https://www.linkedin.com/in/matthew-web-dev/
Email: [abdelmanan.abdelrahman03@gmail.com](mailto:abdelmanan.abdelrahman03@gmail.com)
Resume: [Resume.pdf](https://github.com/vercel/next.js/files/7560579/Matthew.pdf)

**Comment by chankruze** [comment]
**Location:** Bhubaneswar, Odisha, India
**Remote:** Yes
Willing to relocate: Inside India - No, Foreign - Yes
Résumé/CV: https://drive.google.com/drive/u/0/folders/1JMA8rzlZ0aPZhN4hEXzNMPWuuBq56FWS
Email: chankruze@gmail.com
LinkedIn: https://www.linkedin.com/in/chankruze/

**Technologies**

|  Key |  Value | 
|---|---|
| Frontend | React/Nextjs, JavaScript/TypeScript, Tailwind CSS/Chakra UI | 
| Backend | NodeJS, PHP, Go, Graphql, REST | 
| Database | MongoDB, MySQL, SQLite Firebase  |  
| ORM| Prisma | 
| Payment Gateway| AWS, Google Cloud, Vercel  | 
| Cloud Services| Jest, Mocha, Storybook  |  
| Database |  WordPress, Wix.com, Sanity |  
| Languages|  JS/TS, Go, JAVA, C/C++, Python

**Comment by ImBIOS** [comment]
Location: Indonesia
Remote: Yes
Willing to relocate: Saudi Arabia/UAE - Yes, Other - No
Certificates: Google Associate Cloud Engineer
Résumé/CV: https://drive.google.com/file/d/18i0-sBiK3Yo7i3v7iH-oc52ZrYBAqKVX/view?usp=share_link
Website: https://imam.dev
Email: hi@imam.dev

**Comment by eoluwakayode34** [comment]
Location: Nigeria
Remote: Yes
Willing to relocate: Yes
Technologies: NextJs, React
Résumé/CV:
Website:
Email:eoluwakayode34@gmail.com

**Comment by Neo-Ciber94** [comment]
**Location:** Dominican Republic

**Remote**: Yes

**Willing to Relocate**: Yes

**Technologies**:
- NextJS
- NodeJS
- C#
- Rust
- React
- Angular
- MySQL
- SQL Server
- Docker

Résumé/CV: [Link to Résumé/CV](https://d1li3xvarp5pex.cloudfront.net/)

**Comment by PMLyf** [comment]
Location: Jacksonville, FL - USA
Remote: Yes
Willing to relocate: No
Technologies: Javascript, Typescript, React, Next, CSS, Node/Express, Java, Kotlin, Firebase / GCP, MUI, Docker, CI/CD
Résumé/CV: https://docs.google.com/document/d/e/2PACX-1vRBi2rE1LI2VGah_N4zHT3LpjFAjijc-i9ViYFAlEHmBWVVKzYC5mr_IanJarxrNmVO7Ehde6G_vGi8/pub
Website: https://sandideveloper.com/
Email: [david@sandideveloper.com](mailto:david@whatever.management)

**Comment by modecode22** [comment]
Location: Algeria 
Remote: Yes
Willing to relocate: Maybe
Technologies: JS,TS, React, Next, Tailwind, CSS, Node/Express/NEST
Website: https://moncef.net
Email: [aissaouimoncefdev@gmail.com](mailto:aissaouimoncefdev@gmail.com)

**Comment by isaacrundev** [comment]
Location: Vancouver, Canada
Remote: Yes
Willing to relocate: Maybe
Technologies: MERN stack, Next.js, TailwindCSS, MaterialUI, Shadcn/ui, PostgreSQL
Résumé/CV: https://profile.indeed.com/p/isaacw-01mjtzw
Website: https://portfolio-sigma-ten-72.vercel.app/
Email: isaac.run.dev@gmail.com

**Comment by 4Furki4** [comment]
Location: Turkey, Istanbul
Remote: Yes
Willing to relocate: Yes
Technologies: TypeScript, Next.js, MERN Stack, Angular Prisma, PostgreSQL, Tailwind CSS, .NET 7, C#
Résumé/CV: https://furki.vercel.app/CV.pdf
Website:https://furki.vercel.app
Email: muhammedcengiz1@gmail.com

**Comment by emilkovacevic** [comment]
Location :  Serbia
Remote : Yes
Willing to relocate : Yes
Technologies : Next,js, React.js ,Express.Js, Node.Js , .NET, MongoDB, PostgresQL, Tailwind , Git, Jira, ...
[Resume](https://emilthedev.com/about)
Website : [https://emilthedev.com](https://emilthedev.com/about)
Email : [emil.kovacevic.dev@gmail.com](mailto:emil.kovacevic.dev@gmail.com)
